### PR TITLE
feat: add base image for batch execution

### PR DESCRIPTION
**Pull request by TaoSunVoyage**
#172

### CODE REVIEW

File: .github/workflows/acceptance_tests.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+name: Acceptance tests
```
Comment (rokroskar) [inline .github/workflows/acceptance_tests.yml:1]: I think you need to rebase your branch on master...?

**Review by rokroskar — COMMENTED** [review]
Looks good at first glance, but can you please rebase this branch on master so it's easier to see the new changes?

File: .github/workflows/acceptance_tests.yml (hunk 1 above)
Comment (TaoSunVoyage) [reply .github/workflows/acceptance_tests.yml]: Yes! Done.

**Comment by rokroskar** [comment]
ahh much better  🙌

File: docker/batch/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ENV PIPX_BIN_DIR /share/bin
+ENV PIPX_HOME /share/pipx
+ENV PATH /share/bin:$PATH
+
+# install renku-python
+RUN pipx install --pip-args="--no-cache" renku
+
+# inject entrypoint.sh
+COPY --from=renku_base /entrypoint.sh /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
```
Comment (rokroskar) [inline docker/batch/Dockerfile:34]: newline
```suggestion
ENTRYPOINT ["/entrypoint.sh"]

```

**Review by rokroskar — APPROVED** [review]
Thanks!

**Pull request merged**